packages/server/src/common/server.ts

```diff
@@ -30,26 +30,22 @@ class ExpressServer {
         const clientRoot = path.normalize(__dirname + '/../../../client');
         app.set('appPath', root + 'client');
 
+        // logging, should be first middleware
         app.use(morgan('dev'));
 
-        app.use(
-            bodyParser.json({ limit: config.requestLimit })
-        );
-        app.use(
-            bodyParser.urlencoded({
-                extended: true,
-                limit: config.requestLimit,
-            })
-        );
-        app.use(
-            bodyParser.text({ limit: config.requestLimit })
-        );
+        app.use(poweredByMiddleware);
         app.use(cookieParser(process.env.SESSION_SECRET));
-        app.use(express.static(`${root}/public`));
-        app.use(express.static(`${clientRoot}/build`));
+
+        app.use('/static', express.static(`${clientRoot}/build/static`, {
+            maxAge: '30d',
+            immutable: true,  // these files never change and can be cached for 1y
+        }));
+        app.use(express.static(`${clientRoot}/build`, {
+            maxAge: '1m', // index.html should be revalidated much more frequently
```

Comment:
this tells browser to revalidate with the origin every 1m. cra recommends that this should be set to 0 (force revalidation every request) but i think this low ttl will help shed load during a ddos on `/`